### PR TITLE
ADD: byte representation

### LINKED ISSUE #247: Byte Representation for files
image
 Especially for larger files.

### PR DESCRIPTION
**Pull request by realemx**
Closes #247

### CODE REVIEW

File: src/main/java/mops/persistence/file/FileInfo.java [hunk 1]
```diff
@@ -100,11 +109,31 @@ public Instant getLastModifiedTime() {
...
+
+        double result;
+        String suffix;
+
+        if (this.size < kilobyte) {
+            result = this.size;
+            suffix = " B";
+        } else if (this.size > kilobyte && this.size < megabyte) {
+            result = this.size / kilobyte;
+            suffix = " Kb";
```
Comment (Thabb) [inline src/main/java/mops/persistence/file/FileInfo.java]:
```suggestion
            suffix = " KB";
```
Kb = Kilobit
KB = Kilobyte

File: src/main/java/mops/persistence/file/FileInfo.java [hunk 2]
```diff
@@ -100,11 +109,31 @@ public Instant getLastModifiedTime() {
...
+        } else if (this.size > megabyte && this.size < gigabyte) {
+            result = this.size / megabyte;
+            suffix = " Mb";
```
Comment (Thabb) [inline src/main/java/mops/persistence/file/FileInfo.java]:
```suggestion
            suffix = " MB";
```

File: src/main/java/mops/persistence/file/FileInfo.java [hunk 3]
```diff
@@ -100,11 +109,31 @@ public Instant getLastModifiedTime() {
...
+        } else {
+            result = this.size / gigabyte;
+            suffix = " Gb";
```
Comment (Thabb) [inline src/main/java/mops/persistence/file/FileInfo.java]:
```suggestion
            suffix = " GB";
```

File: src/main/java/mops/persistence/file/FileInfo.java [hunk 4]
```diff
@@ -100,11 +109,31 @@ public Instant getLastModifiedTime() {
...
      *
-     * @return FileInfoBuilder
+     * @return Filesize with prefix.
      */
-    public static FileInfoBuilder builder() {
-        return new FileInfoBuilder();
+    public String getSizeString() {
+        final double kilobyte = 1024;
+        final double megabyte = kilobyte * 1024;
+        final double gigabyte = megabyte * 1024;
```
Comment (Thabb) [inline src/main/java/mops/persistence/file/FileInfo.java]:
```suggestion
        final double kilobyte = 1000;
        final double megabyte = kilobyte * 1000;
        final double gigabyte = megabyte * 1000;
```
1000 is used for Kilobyte, Megabyte and Gigabyte and is rounded.
1024 would be used for Kibibyte, Mebibyte and Gibibyte.
https://de.wikipedia.org/wiki/Byte#Vergleichstabelle

File: src/main/java/mops/persistence/file/FileInfo.java [hunk 5]
```diff
@@ -100,11 +109,31 @@ public Instant getLastModifiedTime() {
...
+            result = this.size / kilobyte;
+            suffix = " Kb";
+        } else if (this.size > megabyte && this.size < gigabyte) {
+            result = this.size / megabyte;
+            suffix = " Mb";
+        } else {
+            result = this.size / gigabyte;
+            suffix = " Gb";
+        }
+        return String.format("%.2f" + suffix, result);
```
Comment (iTitus) [inline src/main/java/mops/persistence/file/FileInfo.java]:
```suggestion
        return String.format("%.2f %s", result, suffix);
```

File: src/main/java/mops/persistence/file/FileInfo.java [hunk 6]
```diff
@@ -100,11 +109,31 @@ public Instant getLastModifiedTime() {
...
+        final double kilobyte = 1024;
+        final double megabyte = kilobyte * 1024;
+        final double gigabyte = megabyte * 1024;
+
+        double result;
+        String suffix;
+
+        if (this.size < kilobyte) {
+            result = this.size;
+            suffix = " B";
```
Comment (iTitus) [inline src/main/java/mops/persistence/file/FileInfo.java]:
```suggestion
            suffix = "B";
```

File: src/main/java/mops/persistence/file/FileInfo.java [hunk 7]
```diff
@@ -100,11 +109,31 @@ public Instant getLastModifiedTime() {
...
+        } else if (this.size > kilobyte && this.size < megabyte) {
```
Comment (iTitus) [inline src/main/java/mops/persistence/file/FileInfo.java]:
```suggestion
        } else if (this.size < megabyte) {
```

File: src/main/java/mops/persistence/file/FileInfo.java [hunk 8]
```diff
@@ -100,11 +109,31 @@ public Instant getLastModifiedTime() {
...
+            result = this.size / kilobyte;
+            suffix = " Kb";
+        } else if (this.size > megabyte && this.size < gigabyte) {
```
Comment (iTitus) [inline src/main/java/mops/persistence/file/FileInfo.java]:
```suggestion
        } else if (this.size < gigabyte) {
```

File: src/main/java/mops/persistence/file/FileInfo.java (hunk 4 above)
Comment (iTitus) [reply src/main/java/mops/persistence/file/FileInfo.java]: ???

Comment (iTitus) [reply src/main/java/mops/persistence/file/FileInfo.java]: https://www.youtube.com/watch?v=TLgHOpYBnUM

**Pull request merged**